src/main/java/org/cpsc304/bigdata/db/dao/PatientDAO.java

```diff
@@ -8,12 +8,14 @@
 import java.util.Map;
 
 public interface PatientDAO {
-    Patient findPatientById(final String Id);
+    List<Patient> findPatientById(final String Id);
```

Comment:
Why does this return a list?

Reply:
Because it shares a mapping in the PatientController with another method that returns a list. I didn't want to make another function so I did what you did in the Disease example and just changed the DAO method to return a list.